<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_1{
	background-color: transparent;
	background-image: url(:empty.png);
	background: none;
	background-repeat: none;
	
}
#pushButton_1:hover:!pressed {
	border-radius: 15px; 
	border-style: solid;
	border-width: 3px;
	border-color: black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2{
	background-color: transparent;
	background-image: url(:empty.png);
	background: none;
	background-repeat: none;
	
}
#pushButton_2:hover:!pressed {
	border-radius: 15px; 
	border-style: solid;
	border-width: 3px;
	border-color: black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_3{
	background-color: transparent;
	background-image: url(:empty.png);
	background: none;
	background-repeat: none;
	
}
#pushButton_3:hover:!pressed {
	border-radius: 15px; 
	border-style: solid;
	border-width: 3px;
	border-color: black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_6{
	background-color: transparent;
	background-image: url(:empty.png);
	background: none;
	background-repeat: none;
	
}
#pushButton_6:hover:!pressed {
	border-radius: 15px; 
	border-style: solid;
	border-width: 3px;
	border-color: black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_4{
	background-color: transparent;
	background-image: url(:empty.png);
	background: none;
	background-repeat: none;
	
}
#pushButton_4:hover:!pressed {
	border-radius: 15px; 
	border-style: solid;
	border-width: 3px;
	border-color: black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_5{
	background-color: transparent;
	background-image: url(:empty.png);
	background: none;
	background-repeat: none;
	
}
#pushButton_5:hover:!pressed {
	border-radius: 15px; 
	border-style: solid;
	border-width: 3px;
	border-color: black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_9{
	background-color: transparent;
	background-image: url(:empty.png);
	background: none;
	background-repeat: none;
	
}
#pushButton_9:hover:!pressed {
	border-radius: 15px; 
	border-style: solid;
	border-width: 3px;
	border-color: black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_7{
	background-color: transparent;
	background-image: url(:empty.png);
	background: none;
	background-repeat: none;
	
}
#pushButton_7:hover:!pressed {
	border-radius: 15px; 
	border-style: solid;
	border-width: 3px;
	border-color: black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_8{
	background-color: transparent;
	background-image: url(:empty.png);
	background: none;
	background-repeat: none;
	
}
#pushButton_8:hover:!pressed {
	border-radius: 15px; 
	border-style: solid;
	border-width: 3px;
	border-color: black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="currentplayersymbol">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>173</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Player's X turn.</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentplayersymbol_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayAgainButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Play Again</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LeaveButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>490</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Leave</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
